lib/http/2/compressor.rb

```diff
@@ -444,7 +444,11 @@ def encode(headers)
 
         # Literal header names MUST be translated to lowercase before
         # encoding and transmission.
-        headers.map! { |hk, hv| [hk.downcase, hv] }
+        headers.map! do |hk, hv|
```

Comment:
Hey @HoneyryderChuck is there a corresponding test/spec for this new behaviour please?

Reply:
@Tr4pSt3R not until there is agreement that this is a relevant feature.